quarkchain/evm/messages.py

```diff
@@ -137,6 +136,39 @@ def validate_transaction(state, tx):
     if not tx.sender:  # sender is set and validated on Transaction initialization
         raise UnsignedTransaction(tx)
 
+    if tx.version == 2:
+        chain_config = state.qkc_config.CHAINS[tx.from_chain_id]
```

Comment:
Put a note about how an EIP155 tx is mapped to our tx, and
where is the mapping checked?